Resources/Prototypes/Entities/Structures/Machines/crew_monitor_server.yml

```diff
@@ -28,7 +28,7 @@
       receiveFrequencyId: SuitSensor
       autoConnect: false
     - type: WirelessNetworkConnection
-      range: 500
+      range: 10000 # Delta-V Maximises the range for the Crew Monitor Server to make it work for Listening Post
```

Comment:
Am I missing something or will this have side effects for other things? Why was the range limited in the first place, and what consequences for say, other antags, could increasing it have?

Reply:
The antag-sided side effect of this would be that; should Nukies get their hands on a crew monitor board, they can see who's dead or alive on the station from their ship and with a mass scanner/radar nearby and a few minutes they could vaguely triangulate their location.

When Listening Post first came out; no matter the range I put it at the syndicate server onboard would pick up the sensors and transmit it to the Crew Monitor aboard the listening post. Doesn't seem to work anymore, hence I've had to crank up the range on the server on the actual station.